Appveyor wheels

dougthor42 opened the pull request:
This is a redo of #25 based on the comments therein.

Fixes #24.

I've added Py24 and the others back. I've also made it so wheels are only built on on two versions of Python (this will speed up appveyor build times). All that's left is to modify the wheel script to upload to PyPI.

Please consider merging this, as the wheels make windows installs **significantly** easier.

Scondo commented:
As I wrote before source-only wheels are not good as it disable speedup even on systems with C compiler.

You should add installation of Cython to provide truly binary wheels. I could do it myself later, but I need time for understand 'detect binary' and priority mechanism of building and installing wheels.

Until then `easy_install purepng` provide pure-python mode for windows.

dougthor42 commented:
Alright, I've added Cython to the build steps and changed things so that all versions >= 2.7 build true binary wheels for windows. I'm not going to worry about wheels for anything earlier than 2.7 since those have long been EoL'd.

Does that solve your concerns?

Scondo commented:
OK. I'm not sure if I'll apply this for 0.2.1 but I definitely use this for 0.3.

I'll try to return to this project and release 0.3.0 as soon as possible.
Nothing inspires like good pull request - thank you!
